.devcontainer/bazel-zen/Dockerfile

```
@@ -0,0 +1,81 @@
+
+ARG BASE_IMAGE=focal
+
+FROM mcr.microsoft.com/devcontainers/base:${BASE_IMAGE}
+
+ARG NAME=bazel-zen
+
+################################################################################
+# Unset gcr configured ubuntu user
+################################################################################
+
+USER root
+
+################################################################################
+# Tools & Utilities
+################################################################################
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    # development
+    curl \
+    bash \
+    g++ \
+    gcc \
+    git \
+    graphviz \
+    make \
+    pkg-config \
+    ssh \
+    wget \
+    vim \
+    less \
+    zip \
+    # python
+    python3-dev \
+    && apt-get autoremove -y && apt-get clean -y && rm -rf /var/lib/apt/lists/*
+
+################################################################################
+# Bazelisk
+################################################################################
+
+ARG BAZEL_VERSION=6.1.1
+ARG BAZELISK_VERSION=v1.10.1
+ARG BAZELISK_DOWNLOAD_SHA=dev-mode
+RUN curl -fSsL -o /usr/local/bin/bazelisk https://github.com/bazelbuild/bazelisk/releases/download/${BAZELISK_VERSION}/bazelisk-linux-amd64 \
+    && ([ "${BAZELISK_DOWNLOAD_SHA}" = "dev-mode" ] || echo "${BAZELISK_DOWNLOAD_SHA} */usr/local/bin/bazelisk" | sha256sum --check - ) \
+    && chmod 0755 /usr/local/bin/bazelisk
+ENV USE_BAZEL_VERSION "${BAZEL_VERSION}"
+
+################################################################################
+# Login Shell
+################################################################################
+
+ENV TERM xterm-256color
+ENTRYPOINT ["/bin/bash", "--login", "-i"]
+
+################################################################################
+# User 'zen'
+################################################################################
+
+ARG USERNAME=zen
+
+# Automatically creates a UID, group ${USERNAME}, adds ${USERNAME} to the group
+#
+# NB: If using this image as a base for a devcontainer, it is critical to have
+# just one user, since dynamic UID mapping will not work otherwise, refer to:
+#   https://github.com/microsoft/vscode-remote-release/issues/1155
+#
+RUN useradd -o --uid 1000 -s "/bin/bash" -m ${USERNAME} && \
+    apt-get install -y sudo && \
+    echo "${USERNAME} ALL=NOPASSWD: ALL" > /etc/sudoers.d/${USERNAME} && \
+    chmod 0440 /etc/sudoers.d/${USERNAME}
+
+RUN echo "export PS1='\[\033[01;36m\](docker)\[\033[00m\] \[\033[01;32m\]\u@${NAME}\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '" >> /home/${USERNAME}/.bashrc && \
+    echo "alias ll='ls --color=auto -alFNh'" >> /home/${USERNAME}/.bashrc && \
+    echo "alias ls='ls --color=auto -Nh'" >> /home/${USERNAME}/.bashrc && \
+    echo "alias bazel='bazelisk'" >> /home/${USERNAME}/.bashrc
+
+ENV BLESSING="May you be blessed by a tickle from his noodly apppendages..."
```

nit: 

```suggestion
ENV BLESSING="May you be blessed by a tickle from his noodly appendages..."
```

PS: :smile: